tests/UI/pages/BO/design/themeAndLogo/advancedCustomization/index.js

```diff
@@ -0,0 +1,90 @@
+require('module-alias/register');
+const themeAndLogoBasePage = require('@pages/BO/design/themeAndLogo/themeAndLogo/themeAndLogoBasePage');
+
+/**
+ * Advanced customization page, contains functions that can be used on the page
+ * @class
+ * @extends themeAndLogoBasePage
+ */
+class AdvancedCustomization extends themeAndLogoBasePage {
+  /**
+   * @constructs
+   * Setting up texts and selectors to use on advanced customization page
+   */
+  constructor() {
+    super();
+
+    this.pageTitle = 'Advanced Customization •';
+    this.downloadThemeButton = '#download_child_theme';
+    this.uploadChildThemeModal = '#upload-child-modal';
+    this.uploadChildThemeButton = `#psthemecusto a[data-target= '${this.uploadChildThemeModal}']`;
```

Comment:
I don't think the data-target is related to the selector id mentionned before (even if it's the same thing), So I prefer to keep them separated.
```suggestion
    this.uploadChildThemeButton = `#psthemecusto a[data-target=\'#upload-child-modal\']`;
```